app/helpers/records_helper.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module RecordsHelper
+  def record_kinds
+    {
+      login: Login.model_name.human(count: 0),
+      fail:  Fail.model_name.human(count: 0)
+    }
+  end
+
+  def record_user_name record
+    record.user.present? ? record.user.username : record.data['user_name']
+  end
+
+  def data_translate record
```

Comment:
```suggestion
  def record_human_data_names
```
Es una cuasi regla, aunque no _tan_ dura prefijar todos los métodos de un helper con el singular del helper. Esto anotalo y después te cuento bien, pero básicamente se debe a que todos los helpers comparten el namespace, entonces si declarás un método con el mismo nombre en dos helpers, siempre uno "gana" y eso pasa sin un warning. Para tratar de evitarlo hacemos eso.

Reply:
perfecto franco, despues lo charlamos pero clarisimo.